### PR TITLE
fix: update Contrast > State changes > how to test text

### LINKED ISSUE #1558: Contrast > State changes > How to test: Clarify requirement for redundant state indicators
**Describe the bug**
The test instructions need to clarify that when a UI component has multiple (redundant) non-text state indicators, only one of them needs sufficient contrast.

**Current text**
The visual helper for this requirement highlights links, native widgets, and custom widgets in the target page.

1. In the target page, examine each highlighted element to determine whether it can adopt any of the following states:
a. Focused
b. Mouseover
c. Selected
2. Identify any visual (non-text) indicators that communicate:
a. The boundary of the component's clickable area (such as a border or background color)
b. The component's current state (such as a different background color or a checkmark)
3. Use [Accessibility Insights for Windows](https://go.microsoft.com/fwlink/?linkid=2075365) (or if you are testing on a Mac, the [Colour Contrast Analyser](https://developer.paciellogroup.com/resources/contrastanalyser/)) to verify that the visual indicator has a contrast ratio of at least 3:1 against the adjacent background.
Exception: A lower contrast ratio is allowed if either of the following is true:
a. The component is inactive/disabled.
b. The component's appearance is determined solely by the browser.
4. Record your results:
a. Select **Fail** for any instances that do not meet the requirement.
b. Otherwise, select **Pass**. Or, after you have marked all failures, select **Pass unmarked instances**.

**Propsosed text**
The visual helper for this requirement highlights links, native widgets, and custom widgets in the target page.
1. In the target page, examine each highlighted element to determine whether it supports any of the following states:
a. Focused
b. Mouseover
c. Selected
2. In each supported state, use [Accessibility Insights for Windows](https://go.microsoft.com/fwlink/?linkid=2075365) (or the [Colour Contrast Analyser](https://developer.paciellogroup.com/resources/contrastanalyser/) if you are testing on a Mac) to verify that the following visual presentations (if implemented) have a contrast ratio of at least 3:1 against the adjacent background:
a. Any visual boundary that indicates the component's clickable area
b. Any visual effect that indicates the component's current state.
Note: If an element has redundant state indicators (such as a unique background color _and_ a unique text style), only one indicator is required to have sufficient contrast.
Exception: A lower contrast ratio is allowed if either of the following is true:
a. The component is inactive/disabled.
b. The component's appearance is determined solely by the browser.
4. Record your results:
a. Select **Fail** for any instances that do not meet the requirement.
b. Otherwise, select **Pass**. Or, after you have marked all failures, select **Pass unmarked instances**.

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Update hot to test text according to #1558 

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #1558
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [x] Added a suitable semantic tag to PR title (fix, chore, feat., refactor). Check workflow guide at: `<rootDir>/docs/workflow.md` 
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by Shobhit1** [comment]
Tagging @ferBonnin for the text changes!

File: src/assessments/contrast/test-steps/state-changes.tsx [hunk 1]
```diff
@@ -24,32 +24,29 @@ const howToTest: JSX.Element = (
...
             </li>
             <li>
-                Identify any visual (non-text) indicators that communicate:
+                In each supported state, use <WindowsContrastCheckerAppLink /> (or the <MacContrastCheckerAppLink /> if you are testing on a
+                Mac) to verify that the following visual presentations (if implemented) have a contrast ratio of at least 3:1 against the
+                adjacent background:
                 <ol>
-                    <li>The boundary of the component's clickable area (such as a border or background color)</li>
-                    <li>The component's current state (such as a different background color or a checkmark)</li>
+                    <li>Any visual boundary that indicates the component's clickable area</li>
```
Comment (ferBonnin) [inline src/assessments/contrast/test-steps/state-changes.tsx]: nit, bug from the spec: its missing a "." at the end

Comment (smoralesd) [reply src/assessments/contrast/test-steps/state-changes.tsx]: updated

File: src/assessments/contrast/test-steps/state-changes.tsx [hunk 2]
```diff
@@ -24,32 +24,29 @@ const howToTest: JSX.Element = (
...
+                    <li>Any visual effect that indicates the component's current state.</li>
+                </ol>
+                Note: If an element has redundant state indicators (such as a unique background color and a unique text style), only one
```
Comment (ferBonnin) [inline src/assessments/contrast/test-steps/state-changes.tsx]: nit, the _and_ should be in <em>.

Comment (smoralesd) [reply src/assessments/contrast/test-steps/state-changes.tsx]: updated using `<Markup.Emphasis>`

**Pull request merged**